### PR TITLE
Events app

### PR DESCRIPTION
**Pull request by HKiwooma**
Description of Task to be completed?
Test should check for validity of the username, email and password

What are the relevant pivotal tracker stories?

[160192468](https://www.pivotaltracker.com/story/show/160192468)

### CODE REVIEW

**Comment by Kasulejoseph** [comment]
your tests are really good.. please just  resolve your conflict and every thing will be perfect

File: reception/reception.py [hunk 1]
```diff
@@ -1,37 +1,35 @@
 '''
```
Comment (mozzy22) [inline reception/reception.py:1]: you should start implementing OOP

File: reception/reception.py [hunk 2]
```diff
@@ -1,37 +1,35 @@
...
-# if any(User_Name in mystring for mystring in vip_list) == True:
-#     print ("\n".join(names for names in vip_list if User_Name.title() in names))
-
-def registration_checker(User_Name):
-    if any(User_Name in mystring for mystring in vip_list) == True:
+def registration_checker():
+    User_Name = (input("Please Enter your Name:\n > ")).title()
+    # Ask a user to enter one of their names and check through the lists to see
+    ordinary_list = read_file('ordinary_list.txt')
+    vip_list = read_file('vip_list.txt')
```
Comment (mozzy22) [inline reception/reception.py]: you code looks too compacted , try adding necesary line space as PEP statndards

File: reception/reception.py [hunk 3]
```diff
@@ -1,37 +1,35 @@
...
+    if any(User_Name in mystring for mystring in vip_list) == any(User_Name in mystring for mystring in ordinary_list):
+        print ("\n".join(names for names in vip_list if User_Name.title() in names) + ", VIP")
+        print ("\n" .join(names for names in ordinary_list if User_Name.title() in names) + ", Ordinary")
+    elif any(User_Name in mystring for mystring in vip_list) == True:
```
Comment (mozzy22) [inline reception/reception.py]: u have un necesarily over nested the if ...i know theres a better way to summarise it up with a few ifs

**Pull request merged**